catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/QueryMetacardApplication.java

```diff
@@ -64,6 +64,12 @@
 
   private static final String COUNT = "count";
 
+  private static final String SORT_BY = "sort_by";
+
+  private static final String ATTR = "attr";
+
+  private static final String ASCENDING = "asc";
```

Comment:
❓ Is there anywhere we can share constants for these so it's easier to maintain?

Reply:
I'm not familiar with such a place where we're sharing constants like this for our endpoints.